pkg/controller/migplan/validation.go

```diff
@@ -43,6 +43,7 @@ const (
 	NsNotFoundOnDestinationCluster             = "NamespaceNotFoundOnDestinationCluster"
 	NsLimitExceeded                            = "NamespaceLimitExceeded"
 	NsLengthExceeded                           = "NamespaceLengthExceeded"
+	NsHaveNodeSelectors                        = "NameSpacesHaveNodeSelectors"
```

Comment:
`NameSpacesHaveNodeSelectors ` should be `NamespacesHaveNodeSelectors`

Reply:
Done!